lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -241,15 +243,15 @@ def get(self, request, *args, **kwargs):
 
         data = {
             'course_blocks': course_blocks,
-            'course_expired_html': course_expired_html,
+            'course_expired_html': course_expired_html or None,
```

Comment:
What will it be if not None? Is this an `'' or None`?

Reply:
`False` because of my one-liners like `show_enrolled or get_html`